x-pack/plugins/security_solution/cypress/integration/detection_rules/bulk_edit_rules.spec.ts

```diff
@@ -73,6 +74,13 @@ const customRule = {
   name: RULE_NAME,
 };
 
+const customRuleWithDataViewId = {
```

Comment:
I believe this should be revisited. We are passing index patterns and a data view when it is supposed to be one thing or the other.